src/ViewModel/Factory/PictureBuilderFactory.php

```diff
@@ -50,12 +50,9 @@ public function forImage(Image $image, int $defaultWidth, int $defaultHeight = n
             $type = 'image/jpeg';
         }
 
-        if ($defaultWidth > $image->getWidth()) {
+        if ($defaultWidth > $image->getWidth() && null === $defaultHeight) {
```

Comment:
Either an image is being requested at a specific width (eg article figures), or at an exact size (eg on the about pages). For the former, we can just cap the image at the max width. For the latter, we might have to let the image stretch (if the source is smaller than the requested size).